### PR TITLE
Feature: Generate markdown document from proto file

### PR DESCRIPTION
**Pull request by fr0stf0x**
ref https://github.com/zalopay-oss/backstage-grpc-playground/issues/3

- Install https://github.com/timostamm/protobuf-ts for auto install `protoc` binary
- Support config to install https://github.com/pseudomuto/protoc-gen-doc
  https://github.com/zalopay-oss/backstage-grpc-playground-backend/blob/dceba9498ba0c439404ee36db3928bf2d8e561ad/config.d.ts#L12-L15
- Generate markdown document from proto file

TODO: Cache generated files

### CODE REVIEW

File: src/api/importProtos.ts [hunk 1]
```diff
@@ -84,7 +87,8 @@ export async function getProtosFromEntitySpec(
       imports: imports.map(pSaveProtoTextAsFile),
     };
   } catch (err) {
-    console.log('OUTPUT ~ getProtosFromEntitySpec ~ err', err);
+    logger.info(`Error getProtosFromEntitySpec`);
```
Comment (whatvn) [inline src/api/importProtos.ts:90]: Should this 2 lines combine to one with `logger.error` only, info and error log in 2 lines is confusing

File: src/api/importProtos.ts [hunk 2]
```diff
@@ -192,8 +197,8 @@ export async function loadProtosFromFile(
         try {
           protoDoc = await genDoc(absoluteFilePath, allImports, genDocConfig);
         } catch (err) {
-          console.error(`Error generating document. Please submit a new issue at ${REPO_URL}`);
-          console.error(`Error:`, err);
+          logger.warn(`Error generating document. Please submit a new issue at ${REPO_URL}`);
+          logger.error(err);
```
Comment (whatvn) [inline src/api/importProtos.ts:201]: same with previous comment

File: src/service/router.ts [hunk 3]
```diff
@@ -355,7 +358,15 @@ export async function createRouter(
           }
         }
 
-        const loadProtoResult = await loadProtos(UPLOAD_PATH, filesWithImports, genDocConfig);
+        let isGenDoc = req.body.isGenDoc;
+
+        try {
+          isGenDoc = JSON.parse(isGenDoc);
```
Comment (whatvn) [inline src/service/router.ts:364]: should this just be JSON.parse(req.body.isGenDoc) ?

Comment (fr0stf0x) [reply src/service/router.ts:364]: I wrote it for prevent the case `req.body.isGenDoc` is undefined and JSON.parse will throw error

File: src/api/importProtos.ts (hunk 2 above)
Comment (fr0stf0x) [reply src/api/importProtos.ts:201]: I will fix it

File: src/api/importProtos.ts (hunk 1 above)
Comment (fr0stf0x) [reply src/api/importProtos.ts:90]: I will fix it

**Pull request merged**